components/core/src/crypto/keys.rs

```diff
@@ -309,15 +309,8 @@ fn mk_key_filename<P, S1, S2>(path: P, keyname: S1, suffix: S2) -> PathBuf
 /// generates a revision string in the form:
 /// `{year}{month}{day}{hour24}{minute}{second}`
 /// Timestamps are in UTC time.
-fn mk_revision_string() -> Result<String> {
-    let now = time::now_utc();
-    // https://github.com/rust-lang-deprecated/time/blob/master/src/display.rs
-    // http://man7.org/linux/man-pages/man3/strftime.3.html
-    match now.strftime("%Y%m%d%H%M%S") {
-        Ok(result) => Ok(result.to_string()),
-        Err(_) => Err(Error::CryptoError("Can't parse system time".to_string())),
-    }
-}
+// TODO (CM): Return a String, not a Result
+fn mk_revision_string() -> Result<String> { Ok(time::OffsetDateTime::now().format("%Y%m%d%H%M%S")) }
```

Comment:
Would it be possible to use `chrono`'s [`format`](https://docs.rs/chrono/0.4.10/chrono/struct.DateTime.html#method.format) here and then we could be completely rid of `time`?

Reply:
:thinking: Hrmm, that might be worth looking into. `chrono` actually wraps `time`, but maybe we could standardize on `chrono` to help simplify things; we do use `chrono` elsewhere, after all.

Reply:
Actually, yes. We use `chrono` in just a small handful of places in the rest of the code, but it's always around formatting dates. I think it makes a lot of sense to standardize on `chrono` for that kind of stuff (Or standardize on `time`, for that matter!)

Reply:
OK, we're on chrono, now!